layouts/partials/article.html

```diff
@@ -5,7 +5,9 @@
 				</header>
 {{ end }}
 {{ if or (eq $.mode "full") (eq $.mode "body") }}
+				{{ if default (default true (index .Site.Params "toc")) (index .Params "toc") }}
 				{{ .TableOfContents }}
```

Comment:
The line seemed a bit long with the logic combined into one, so I broke it up like this. Is this alright?